storybook/stories/Layout/Checkbox.tsx

```diff
@@ -14,12 +14,13 @@ function CheckboxItem(): ReactElement {
       onToggle={setIsChecked}
     />
   );
-}
+};
 
 storiesOf("Layout", module).add("Checkbox", () => (
   <View style={{ backgroundColor: color("grey", 10) }}>
-    <CheckboxItem />
-    <CheckboxItem />
+    <View style={{ marginBottom: size(3) }}>
```

Comment:
What's the reason for this addition?

Reply:
putting style to separate the element..previously is without space between element